java/test/jmri/implementation/AbstractTurnoutTestBase.java

```diff
@@ -48,9 +48,9 @@
         @Override
         public void propertyChange(java.beans.PropertyChangeEvent e) {
             listenerResult = true;
-            if(e.getPropertyName().equals("KnownState")){
-		    listenStatus = ((Integer) e.getNewValue()).intValue();
-	    }
+            if (e.getPropertyName().equals("KnownState")) {
+		        listenStatus = ((Integer) e.getNewValue()).intValue();
```

Comment:
Why the odd indentation?

Reply:
It's a class, it was in strange indented before